core/vm/eips.go

```diff
@@ -218,6 +219,17 @@ func opBaseFee(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]
 	return nil, nil
 }
 
+// enable4788 applies EIP-4788 (BEACONROOT opcode)
+func enable4788(jt *JumpTable) {
```

Comment:
Please expand this comment a bit
```suggestion
// enable4788 applies EIP-4788 (BEACONROOT opcode). 
// OBS: it enables it only partially; adding the new opcode. The EIP also requires
// enabling the precompile, which is not handled by this function. 
func enable4788(jt *JumpTable) {
```